easybuild/tools/modules.py

```diff
@@ -833,31 +835,32 @@ def available(self, mod_name=None):
 
     def update(self):
         """Update after new modules were added."""
-        spider_cmd = os.path.join(os.path.dirname(self.cmd), 'spider')
-        cmd = [spider_cmd, '-o', 'moduleT', os.environ['MODULEPATH']]
-        self.log.debug("Running command '%s'..." % ' '.join(cmd))
+        if build_option('update_modules_tool_cache'):
```

Comment:
we've tried to stay away from in-line `return` statements, so I'm not in favour of this...